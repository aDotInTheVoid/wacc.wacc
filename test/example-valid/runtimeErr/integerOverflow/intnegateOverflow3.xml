<unit>
  <main>
    <addr_local>0</addr_local>
    <num>20000</num>
    <neg/>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
    <addr_local>0</addr_local>
    <local>0</local>
    <num>100000000</num>
    <op>mul</op>
    <assign_do/>
    <local>0</local>
    <print><kind>i32</kind><newline>true</newline></print>
  </main>
</unit>