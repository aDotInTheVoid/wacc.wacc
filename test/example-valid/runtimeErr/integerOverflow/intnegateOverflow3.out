-20000
1454759936
